lib/datadog/tracing.rb

```diff
@@ -127,6 +127,13 @@ def log_correlation
         correlation.to_log_format
       end
 
```

Comment:
Because this is the main method in the new public API for baggage, we should add documentation about what it does, from the user perspective. Take a look at the other methods in this class for some good examples.